meorg_client/client.py

```diff
@@ -64,13 +64,16 @@ def _make_request(self, method: str, endpoint: str, data: dict=None, headers: di
 
         # GET/PUT requests have the data interpolated into the url
         if method in ['GET', 'PUT']:
-            endpoint = endpoint.format(data)
+            endpoint = endpoint.format(**data)
+
+        # Remove the leading slash
+        endpoint = endpoint[1:] if endpoint[0] == '/' else endpoint
 
         url = f"{self.base_url}/{endpoint}"
         all_headers = {**self.headers, **headers} if headers else self.headers
         response = requests.request(method, url, data=data, headers=all_headers)
 
-        if response.status_code == 200:
+        if response.status_code in [200, 202]:
```

Comment:
Needed to add additional. response code here. I think it is triggered by the PUT.